massa-async-pool/src/config.rs

```diff
@@ -7,8 +7,10 @@
 pub struct AsyncPoolConfig {
     /// max number of messages in the pool
     pub max_length: u64,
-    /// max async message data (for bootstrap limits)
-    pub max_async_message_data: u64,
+    /// max handler length
+    pub max_function_length: u16,
+    /// max handler params length (for bootstrap limits)
```

Comment:
The "(for bootstrap limits)" is probably no longer relevant here.

Reply:
Done